stubs/jsonschema/jsonschema/_format.pyi

```diff
@@ -8,15 +8,19 @@ class FormatChecker:
     def check(self, instance, format) -> None: ...
     def conforms(self, instance, format): ...
 
-draft3_format_checker: Any
-draft4_format_checker: Any
-draft6_format_checker: Any
-draft7_format_checker: Any
+draft3_format_checker: FormatChecker
+draft4_format_checker: FormatChecker
+draft6_format_checker: FormatChecker
+draft7_format_checker: FormatChecker
+draft201909_format_checker: FormatChecker
+draft202012_format_checker: FormatChecker
 
 def is_email(instance): ...
 def is_ipv4(instance): ...
 def is_ipv6(instance): ...
-def is_host_name(instance): ...
+
+# is_host_name is only defined if fqdn is installed.
+# def is_host_name(instance): ...
```

Comment:
We prefer false positives over false negatives, so it's better to leave the comment, but leave the definition itself uncommented, so that users that have fqdn installed don't get errors.

Reply:
That's what I had before, but the stubtests were failing because `is_host_name` was not defined during runtime. Should I tell the test kit to install all optional dependencies during testing?

Reply:
You can add entries to `stubs/jsonschema/@tests/stubtest_allowlist.txt` to prevent the failure.